src/wallet.rs

```diff
@@ -206,21 +202,23 @@ pub async fn get_balance(
 mod tests {
     use super::*;
     use ctor::ctor;
-    use dotenv::dotenv;
+    use dotenv;
 
     #[cfg(test)]
     #[ctor]
     fn init() {
-        dotenv().ok();
+        dotenv::from_filename(".env.example").ok();
```

Comment:
Actually the correct usage is:

`cp .env.example .env`

The dotenv module use `.env` by default.

Reply:
we don't want additional side effect from test.  
leave it there so that we won't overwrite the user config.

Reply:
right, for tests we definitely want this.